File: Siddhi mam Tasks/Day 23/Java/MultiScreen_2/app/src/main/java/com/example/multiscreen_2/MainActivity2.java
package com.example.multiscreen_2;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

public class MainActivity2 extends AppCompatActivity {

    TextView z ;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main2);

        z = findViewById(R.id.textView2);

        Intent ma = getIntent();

    String result = ma.getStringExtra(MainActivity.MEME);


     z.setText("your age is " + result);


    }
}

File: Siddhi mam Tasks/Day 38/Model passing in Intent (Arraylist)/app/src/main/java/com/example/mock_review/MainActivity2.kt
package com.example.mock_review

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView

class MainActivity2 : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main2)

        // Find the TextView
        val textView = findViewById<TextView>(R.id.textView)

        val modelList = intent.getSerializableExtra("modelList") as ArrayList<MyModel>?

        // Displaying data
        modelList?.let {
            for (model in it) {
                textView.append("Name: ${model.name}, Age: ${model.age}\n")
            }
        }
    }
}